ProjectLighthouse/Configuration/EmailEnforcementConfiguration.cs

```diff
@@ -0,0 +1,72 @@
+#nullable enable
+using System.Collections.Generic;
+using YamlDotNet.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Configuration;
+
+public class EmailEnforcementConfiguration : ConfigurationBase<EmailEnforcementConfiguration>
+{
+    public override int ConfigVersion { get; set; } = 1;
+
+    public override string ConfigName { get; set; } = "enforce-email.yml";
+
+    public override bool NeedsConfiguration { get; set; } = false;
+
+    public bool EnableEmailEnforcement { get; set; } = false;
+    public bool EnableEmailBlacklist { get; set; } = false;
```

Comment:
```suggestion
    public bool EmailBlacklistEnabled { get; set; } = false;
```
to stay consistent with other config files